testUtils.ts

```diff
@@ -8,6 +8,20 @@ export function getByClass(
   return wrapper.get(`.${className}`)
 }
 
+export function getById(
+  wrapper: VueWrapper<any>,
+  testId: string,
+): WrappedElement {
+  return wrapper.get(`#${testId}`)
+}
+
+export function getByQuery(
+  wrapper: VueWrapper<any>,
+  query: string,
+): WrappedElement {
+  return wrapper.get(query)
+}
+
```

Comment:
I'm not entirely sure that `VueWrapper` is the correct type. Do me a favor and double-check that that's actually what `mountSuspended` returns.

Reply:
Did a look into the Types

This is the type of the `mountSuspended` 
```ts
const wrapper: VueWrapper<Partial<{
    modal: boolean;
}> & Omit<{
    readonly modal: boolean;
    onClose?: (() => any) | undefined;
    onForgotPasswordClick?: (() => any) | undefined;
    onSignUpClick?: (() => any) | undefined;
} & ... 4 more ... & {
    ...;
}, "modal"> & ... 5 more ... & {}, ComponentPublicInstance<...>> & {
    ...;
}
```

Reply:
Technically we can do better typing here but `VueWrapper<any>` is fine.